packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -692,4 +707,61 @@ export class SDKClient {
 
     return fileId;
   };
+
+  /**
+   * @dev Deletes `fileId` file from the Hedera Network utilizing Hashgraph SDK client
+   * @param client
+   * @param fileId
+   * @param requestId
+   * @param callerName
+   * @param interactingEntity
+   */
+  private deleteFile = async (
+    client: Client,
+    fileId: FileId,
+    requestId?: string,
+    callerName?: string,
+    interactingEntity?: string,
+  ) => {
+    // format request ID msg
+    const requestIdPrefix = formatRequestIdMessage(requestId);
+
+    try {
+      // Create fileDeleteTx
+      const fileDeleteTx = await new FileDeleteTransaction()
+        .setFileId(fileId)
+        .setMaxTransactionFee(
+          Hbar.fromTinybars(Math.floor((await this.getTinyBarGasFee('FileDelete')) * constants.BLOCK_GAS_LIMIT)),
+        )
+        .freezeWith(client);
+
+      // execute fileDeleteTx
+      const fileDeleteTxResponse = await fileDeleteTx.execute(client);
+
+      // get fileDeleteTx's record
+      const deleteFileRecord = await fileDeleteTxResponse.getRecord(this.clientMain);
+
+      // capture metrics
+      this.captureMetrics(
+        SDKClient.transactionMode,
+        fileDeleteTx.constructor.name,
+        Status.Success,
+        deleteFileRecord.transactionFee.toTinybars().toNumber(),
+        deleteFileRecord?.contractFunctionResult?.gasUsed,
+        callerName,
+        interactingEntity,
+      );
+
+      // ensure the file is deleted
+      const receipt = deleteFileRecord.receipt;
+      const fileInfo = await new FileInfoQuery().setFileId(fileId).execute(client);
+      if (receipt.status === Status.Success && fileInfo.isDeleted) {
+        this.logger.trace(`${requestIdPrefix} Deleted file with fileId: ${fileId}`);
+      } else {
+        throw new SDKClientError({}, `${requestIdPrefix} Fail to delete file with fileId: ${fileId} `);
```

Comment:
Don't throw.
Let's leave this as best effort

Reply:
Same for the catch. Make it no op so that we don't give the wrong response due to cleanup attempts.
Warning logs should be fine